_1327/documents/views.py

```diff
@@ -88,7 +88,7 @@ def edit(request, title, new_autosaved_pages=None, initial=None):
 			'form': form,
 			'attachment_form': attachment_form,
 			'active_page': 'edit',
-			'creation': (len(revisions.get_for_object(document)) == 0),
```

Comment:
thanks, that made me checking it again and now using the same creation check everywhere